Cargo.toml

```diff
@@ -5,20 +5,20 @@ version = "0.2.0"
 [dependencies]
 activitypub = "0.1.3"
 atom_syndication = "0.6"
-canapi = "0.1"
+canapi = "0.2"
 colored = "1.6"
 dotenv = "0.13"
 failure = "0.1"
 gettext-rs = "0.4"
 guid-create = "0.1"
 heck = "0.3.0"
-rpassword = "2.0"
+rpassword = "2.1"
```

Comment:
I think we don't need rpassword anymore in Plume itself (only in the CLI), maybe we should remove it with this PR too?

Reply:
I did it for hyper, there is no reason to not do it for rpassword too